SSDAlignment/ssdalignment_job.fcl

```diff
@@ -2,13 +2,22 @@
 #include "SSDAlignment.fcl"
 
 #include "MakeSSDClusters.fcl"
+#include "ChannelMap.fcl"
+#include "Geometry.fcl"
+#include "DetGeoMap.fcl"
+#include "Align.fcl"
 #include "FillDataQuality.fcl"
 
 process_name: SSDAlignment
 
 services:
 {
   TFileService: { fileName: "ssd_align_hist_r%r_s%04s.root" closeFileFast: false }
+  ChannelMapService: @local::standard_channelmap
```

Comment:
these are all included by the @table::standard_services from [Services.fcl](https://github.com/EMPHATICSoft/emphaticsoft/blob/main/Utilities/Services.fcl) so shouldn't be needed here

Reply:
Also removed these excess lines.